lib/fluent/plugin/in_tail.rb

```diff
@@ -120,26 +119,20 @@ def configure_tag
     def start
       super
 
+      @parser = parser_create(conf: @config.elements('parse').first)
```

Comment:
Hmm... I see. This API design seems not good for users.
configuration error should be found in `configure` phase.
So I recommend to change Parser API, maybe also Formatter, into 2 layers.

```
# configure
@parser = parser_create(conf: @config.elements('parse').first)

# start
@parser.start
```